pipelines/e2e/README.md

```diff
@@ -0,0 +1,17 @@
+# E2E test Pipeline
+
+Tekton pipeline to test the Release Service through E2E.
```

Comment:
Could we maybe add some more details on what the pipeline contains? We probably don't have much details for the other pipelines either, but it would be nice to at least say that it contains just the collect-data and verify-enterprise-contract tasks.

Reply:
I changed it to:
```
Tekton pipeline to test the Release Service through E2E. This Pipeline is the simplest expression of a Release Pipeline. It contains only tasks that could affect or be affected by other services.
```
Do you still want to reference specific tasks?

Reply:
Thanks.

> Do you still want to reference specific tasks?

No, it's fine. But reading the text, it seems there is a negative missing? It contains tasks that could not be affected by other services, right?

Also, could you copy the description to the spec.description as well?

Reply:
I don't think that is missing anything, although it could be rewritten so it's more clear. Changed to:
```
Minimal Tekton Pipeline to be used by the E2E tests. The Pipeline is designed to include only tasks that could be
affected by other services or which results could affect the RHTAP workflow.
```
Updated the spec as well.

Reply:
Hmm, I guess I misunderstood. I thought by other services you mean for example Pyxis and that we don't want to include tasks that rely on Pyxis. But you actually mean other rhtap services, right? Maybe include that? "by other RHTAP services"?

Reply:
Done.

Reply:
Thanks.